src/__test__/persistentSubscription/listPersistentSubscriptions.test.ts

```diff
@@ -0,0 +1,137 @@
+import {
+  createManyPs,
+  createTestNode,
+  delay,
+  jsonTestEvents,
+} from "@test-utils";
+
+import {
+  END,
+  EventStoreDBClient,
+  ROUND_ROBIN,
+  START,
+} from "@eventstore/db-client";
+
+describe("listPersistentSubscriptions", () => {
+  const node = createTestNode();
+  let client!: EventStoreDBClient;
+
+  beforeAll(async () => {
+    await node.up();
+
+    client = new EventStoreDBClient(
+      {
+        endpoint: node.uri,
+      },
+      { rootCertificate: node.rootCertificate },
+      { username: "admin", password: "changeit" }
+    );
+  });
+
+  afterAll(async () => {
+    await node.down();
+  });
+
```

Comment:
**SUGGESTION:** It may be worth checking the case where there are no persistent subscriptions created.

Reply:
Added to all persistent subscription control tests https://github.com/EventStore/EventStore-Client-NodeJS/pull/273/commits/e1bf259d8b7496b13a018241bb915aaf77c997e0